taipy/rest/app.py

```diff
@@ -26,6 +26,12 @@ def create_app(testing=False, flask_env=None, secret_key=None) -> Flask:
         TESTING=os.getenv("TESTING", testing),
         SECRET_KEY=os.getenv("SECRET_KEY", secret_key),
     )
+    if __name__ == "__main__":
```

Comment:
This module will mostly never be the `__main__` module.

Don't use this condition here or at all, unless there is a very specific reason for it.